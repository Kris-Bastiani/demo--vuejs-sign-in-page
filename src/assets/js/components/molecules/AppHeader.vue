<template lang="pug">
	header.header.container.w_max__lrg
		a.logo(href='https://www.frankiefinancial.com/')
			img(src='/assets/images/logo.png', alt='Logo: Frankie')

		div
			a.h2(href='https://www.frankiefinancial.com/') About Frankie
</template>

<style lang="scss">
	.header {
		align-items: center;
		display: flex;
		justify-content: space-between;
		margin: 0 auto 5rem;
		padding: 1rem;

		.logo {
			&,
			> img { display: block; }

			> img { max-width: 13.5rem; }

			@media (max-width: 680px) {
				> img { max-width: 8rem; }
			}
		}
	}
</style>
