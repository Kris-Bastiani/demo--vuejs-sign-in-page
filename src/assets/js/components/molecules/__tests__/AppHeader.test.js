import { shallowMount } from '@vue/test-utils';
import AppHeader from '../AppHeader.vue';

describe('AppHeader', () => {
	it('Matches snapshot', () => {
		const wrapper = shallowMount(AppHeader);
		expect(wrapper.element).toMatchSnapshot();
	});
});
