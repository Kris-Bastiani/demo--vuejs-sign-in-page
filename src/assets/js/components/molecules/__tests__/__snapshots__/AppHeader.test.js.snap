// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader Matches snapshot 1`] = `
<header
  class="app_header container w_max__lrg"
>
  <a
    class="logo"
    href="https://www.frankiefinancial.com/"
  >
    <img
      alt="Logo: Frankie"
      src="/assets/images/logo.png"
    />
  </a>
  <div
    class="flx_aln_y__center"
  >
    <a
      class="h2 dsp__none__below_480"
      href="https://www.frankiefinancial.com/"
    >
      About Frankie
    </a>
    <user-menu-stub
      class="mrg_l__lrg"
    />
  </div>
</header>
`;
